app/app.js

```diff
@@ -25,7 +27,7 @@ import configureStore from './store';
 
 // Observe loading of Open Sans (to remove open sans, remove the <link> tag in
 // the index.html file and this observer)
-import styles from 'containers/App/styles.css';
+import styles from './containers/App/styles.css';
```

Comment:
You're absolutely right. I just carefully read through the webpack and see the `resolve` option in the `webpack.base.babel.js`. Will modify the commit soon :)